CHANGES.rst

```diff
@@ -1,3 +1,189 @@
+   `#2133 <https://github.com/cta-observatory/ctapipe/pull/2133>`__,
+   `#2138 <https://github.com/cta-observatory/ctapipe/pull/2138>`__,
+   `#2217 <https://github.com/cta-observatory/ctapipe/pull/2217>`__,
+   `#2229 <https://github.com/cta-observatory/ctapipe/pull/2229>`__,
+   `#2140 <https://github.com/cta-observatory/ctapipe/pull/2140>`__]
+
+- ``Tool`` now comes with an ``ExitStack`` that enables proper
+  handling of context-manager members inside ``Tool.run``.
+  Things that require a cleanup step should be implemented
+  as context managers and be added to the tool like this:
+
+  .. code::
+
+      self.foo = self.enter_context(Foo())
+
+  This will ensure that ``Foo.__exit__`` is called when the
+  ``Tool`` terminates, for whatever reason. [`#1926 <https://github.com/cta-observatory/ctapipe/pull/1926>`__]
+
+- Implement atmospheric profiles for conversions from h_max to X_max.
+  The new module ``ctapipe.atmosphere`` has classes for the most common cases
+  of a simple ``ExponentialAtmosphereDensityProfile``, a ``TableAtmosphereDensityProfile``
+  and CORSIKA's ``FiveLayerAtmosphereDensityProfile``. [`#2000 <https://github.com/cta-observatory/ctapipe/pull/2000>`__]
+
+- ``TableLoader`` can now also load observation and scheduling block configuration. [`#2096 <https://github.com/cta-observatory/ctapipe/pull/2096>`__]
+
+- The ``ctapipe-info`` tool now supports printing information about
+  the available ``EventSource`` and ``Reconstructor`` implementations
+  as well as io and reco plugins. [`#2101 <https://github.com/cta-observatory/ctapipe/pull/2101>`__]
+
+- Allow lookup of ``TelescopeParameter`` values by telescope type. [`#2120 <https://github.com/cta-observatory/ctapipe/pull/2120>`__]
+
+- Implement a ``SoftwareTrigger`` component to handle the effect of
+  selecting sub-arrays from larger arrays in the simulations.
+  The component can remove events where the stereo trigger would not have
+  decided to record an event and also remove single telescopes from events
+  for cases like the CTA LSTs, that have their own hardware stereo trigger
+  that requires at least two LSTs taking part in an event. [`#2136 <https://github.com/cta-observatory/ctapipe/pull/2136>`__]
+
+
+- It's now possible to transform between ``GroundFrame`` coordinates
+  and ``astropy.coordinates.EarthLocation``, enabling the conversion
+  between relative array coordinates (used in the simulation) and
+  absolute real-world coordinates. [`#2167 <https://github.com/cta-observatory/ctapipe/pull/2167>`__]
+
+- The ``ctapipe-display-dl1`` tool now has a ``QualityQuery`` instance which can be used
+  to select which images should be displayed. [`#2172 <https://github.com/cta-observatory/ctapipe/pull/2172>`__]
+
+- Add a new ``ctapipe.io.HDF5Merger`` component that can selectively merge
+  HDF5 files produced with ctapipe. The new component is now used in the
+  ``ctapipe-merge`` tool but can also be used on its own.
```

Comment:
We could mention its also used in `apply-models`

Reply:
done